OneCommunityGlobal/HGNRest
# Imran toggle invisibility permission self and others

# Description
![Desc](https://github.com/OneCommunityGlobal/HGNRest/assets/87269725/b1f9280f-6223-46ff-a3cb-01817cff3c63)

## Main changes explained:
1. Added toggleInvisibility permission in  src/utilities/createInitialPermissions.js
2. Created a new route in src/routes/userProfileRouter.js
3. Added toggleVisibility function in src/controllers/userProfileController.js

## Related PRS (if any):
To test this backend PR you need to checkout the [PR#2183](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2183)  frontend PR.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. Refer to [PR#2183](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2183) to test
